WcaOnRails/app/views/server_status/index.html.erb

```diff
@@ -58,11 +58,8 @@
     </div>
   </div>
 
-  <div>
-    PID <%= Process.pid %>
-  </div>
+  <%= alert :info, "PID #{Process.pid}" %>
+  <%= alert :info, "Uptime #{duration_to_s(Time.now - WcaOnRails::BOOTED_AT)}" %>
```

Comment:
If it's easy, I like seeing screenshots in PRs that change how things look, since I'm usually too lazy to actually run the code myself.